regulations/templates/regulations/tree/preamble_intro.html

```diff
@@ -55,7 +55,7 @@ <h4 class="small-header">{{ meta.primary_agency }}</h4>
       <a class="citation internal" href="#supporting-docs">View supporting documents from the docket</a>
     </div>
     {% endif %}
-    <hr />
+    <hr class="preamble-divider" />
```

Comment:
More recent definitions of `<hr>` state it to be a thematic break, so like between paragraphs of content. Instead of putting spacing on the `<p>` below, can you put a `margin-bottom`? It would space below a `border-bottom`.

Refs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr
http://html5doctor.com/small-hr-element/